app/views/taxonomies/_form.html.erb

```diff
@@ -99,7 +99,14 @@
 
       <% if User.current.allowed_to?(:view_params) %>
         <div class="tab-pane" id="params">
-          <%= render "common_parameters/parameters", { :f => f, :type => "#{controller_name.singularize}_parameters".to_sym } %>
+          <% taxonomy_type = controller_name.singularize %>
+          <% if taxonomy.parent.present? %>
+              <h4><%= _('Parent parameters') %></h4>
+              <%= render "common_parameters/inherited_parameters", { :inherited_parameters => taxonomy.parent_params(true),
+                                                                     :parameters => taxonomy.send("#{taxonomy_type}_parameters".to_sym)} %>
+          <% end %>
+          <h4><%= _("#{taxonomy_type.capitalize} parameters") %></h4>
```

Comment:
This will not be translated correctly. You need to explicitly have the strings for translations as the taxonomy type also needs to be translated.